test/TestProjects/FirstTest-TypeSpec/tests/Generated/Samples/Samples_FirstTestTypeSpecClient.cs

```diff
@@ -1350,7 +1350,7 @@ public async Task Example_FirstTestTypeSpec_DateTimeBody_ShortVersion_Async()
             Uri endpoint = new Uri("<https://my-service.azure.com>");
             FirstTestTypeSpecClient client = new FirstTestTypeSpecClient(endpoint);
 
-            using RequestContent content = RequestContent.Create("2022-05-10T14:57:31.2311892-04:00");
+            using RequestContent content = RequestContent.Create("2022-05-10T18:57:31.2311892Z");
```

Comment:
is the format change here intended?

Reply:
Yes. Default format for JSON body is RFC3339, default format for header is RFC1123
I have moved it into separate PR: https://github.com/Azure/autorest.csharp/pull/4371